CPlusPlus/Release_Notes.html

```diff
@@ -0,0 +1,853 @@
+<li class="x_xxxxmsonormal">Removes the .NET Documentation from the installer, it can now be found on the website: <a href="https://dev.datalogics.com/apdfl/Documentation/DotNET/Overview/index.html" target="_blank" rel="noopener noreferrer" data-auth="NotApplicable" data-linkindex="3">https://dev.datalogics.com/apdfl/Documentation/DotNET/Overview/index.html</a></li>
+<li class="x_xxxxmsonormal">Removes the  APDFL18_API_Reference.pdf from the release package.</li>
+<li class="x_xxxxmsonormal">Removes outdated documentation files PDF_Library_Overview.docx and PDF_Library_Overview.pdf from the installers.</li>
+<li class="x_xxxxmsonormal">Removes the Working_with_DotNET_6.pdf document in favor of enhanced .NET html documentation.</li>
+<li class="x_xxxxmsonormal">Removes the .NET Samples from the installers, samples for these interfaces should be obtained from <a href="https://github.com/datalogics/apdfl-csharp-dotnet-samples" target="_blank" rel="noopener noreferrer" data-auth="NotApplicable" data-linkindex="4">https://github.com/datalogics/apdfl-csharp-dotnet-samples</a></li>
+<li class="x_xxxxmsonormal">Removes the .NET Framework Samples from the installers, samples for these interfaces should be obtained from <a href="https://github.com/datalogics/apdfl-csharp-dotnet-framework-samples" target="_blank" rel="noopener noreferrer" data-auth="NotApplicable" data-linkindex="5">https://github.com/datalogics/apdfl-csharp-dotnet-framework-samples</a></li>
+<li class="x_xxxxmsonormal">Removes the Java Samples from the installers, samples for these interfaces should be obtained from <a href="https://github.com/datalogics/apdfl-java-maven-samples" target="_blank" rel="noopener noreferrer" data-auth="NotApplicable" data-linkindex="6">https://github.com/datalogics/apdfl-java-maven-samples</a></li>
+<li class="x_xxxxmsonormal">Incorporate new Foxit Conversion SDK into APDFL 18.</li>
+<li class="x_xxxxmsonormal">JAR files are updated to be consistent across all platforms, allowing cross-platform development and reducing crashes.</li>
+<li class="x_xxxxmsonormal">With this release, the Nuget package version has been increased to 18.20.0.</li>
+<li class="x_xxxxmsonormal">With this release, the Maven package version has been increased to 18.20.0.</li>
+</ul>
+<p class="x_xxxxmsonormal">Problem Corrections:</p>
+<ul type="disc">
+<li class="x_MsoListParagraph">SF#45461 - Corrects an issue where Color Conversion could result in hidden content when a matrix contained very large values.</li>
+<li class="x_MsoListParagraph">SF#45471 - Corrects an issue where a Form/BBox with very large values could be ignored during Rendering caused content to be missing in the output.</li>
```

Comment:
Extra B

Reply:
The first B is for Bounding - it's an APDFL shorthand for Bounding Box.